package.json

```diff
@@ -62,7 +62,7 @@
     "esformatter-eol-last": "^1.0.0",
     "esformatter-jsx": "^5.0.4",
     "esformatter-quotes": "^1.0.3",
-    "eslint": "~2.2.0",
+    "eslint": "^2.13.*",
```

Comment:
There's an error about `es-traverse` the fix also requires the `babel-eslint` later in the file. This with the #394, should fix linting.